Orange/data/io.py

```diff
@@ -674,6 +677,9 @@ def write_file(cls, filename, data):
             writer = csv.writer(file, delimiter=cls.DELIMITERS[0])
             cls.write_headers(writer.writerow, data)
             cls.write_data(writer.writerow, data)
+        if isinstance(filename, str) and data.attributes:
+            with open(filename + '.apck', 'wb') as f:
+                pickle.dump(data.attributes, f, pickle.HIGHEST_PROTOCOL)
```

Comment:
Could these two chunks be methods on `FileFormat` class? There might be use for them with other readers/writers?